about.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>À propos</title>
+    <!-- link CSS --> 
+    <link rel="stylesheet" href="css/style.css">
+    <!-- link Boostrap --> 
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+   
+</head>
+<body>
+  <header>
+    <nav class="navbar navbar-expand-lg static-top fixed-top ">
+      <div class="container">
+        <a class="navbar-brand" href="index.html">
+          <img src="img/logo_a.png" alt="Logo portifolio ALine" width="50">
+        </a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ms-auto centered">
+            <li class="nav-item mt-2">
+              <a class="nav-link" id="nav_style" href="index.html">Accueil</a>
+            </li>
+            <li class="nav-item mt-2">
+              <a class="nav-link active" aria-current="page" id="nav_style" href="about.html">À propos</a>
+            </li>
+            <li class="nav-item mt-2">
+              <a class="nav-link" id="nav_style" href="mesprojects.html">Mes Projects</a>
+            </li>
+            <li class="nav-item  ">
+              <a class="nav-link" href="contact.html"><button type="button" class="btn btn-primary btn-outline-primary" id="nav_btn" >Contact</button></a>
+            </li>  
+          </ul>
+      </div>
+    </nav>
+  </header>
+
+      <div class="container mt-3">
+        <h1 id="titre-pages">À propos</h1>
```

Comment:
Sometimes you are using titles with "kebab-case", others with "snake_case". It is good to keep a standard.